src/cubeb_oss.c

```diff
@@ -0,0 +1,1023 @@
+/*
+ * Copyright © 2019-2020 Nia Alarie <nia@NetBSD.org>
+ * Copyright © 2020 Ka Ho Ng <khng300@gmail.com>
+ *
+ * This program is made available under an ISC-style license.  See the
+ * accompanying file LICENSE for details.
+ */
+
+#include <assert.h>
+#include <errno.h>
+#include <sys/types.h>
+# if defined(__FreeBSD__) || defined(__DragonFly__)
```

Comment:
Inconsistent identation, see closing `#endif`.
```suggestion
#if defined(__FreeBSD__) || defined(__DragonFly__)
```